frontend/src/Points.tsx

```diff
@@ -93,9 +94,26 @@ const Points = (props: IProps): JSX.Element => {
 
   thresholdStr += ` (next threshold: ${nextThreshold}分)`;
 
+  const { results: scoreTransitions } = explainScoring({
+    params: props.gameScoringParameters,
+    smaller_landlord_team_size: false,
```

Comment:
ah, probably good to leave a comment here: since you're only looking at point thresholds, the `smaller_landlord_team_size` value doesn't change any of the data that you would care about. But, from the API contract perspective, we actually know whether the landlord has a small team in the parent component.